### PR TITLE
Testing To Do List: part 2

### PR DESCRIPTION
**Pull request by WilhelmK109**
### In this project, I did the following:
- Wrote test code to test functions responsible for the status and content updates.

Screenshot (83)

### CODE REVIEW

**Review by Gambit142 — COMMENTED** [review]
Hi @WilhelmK109 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that the Eslint linters checks failed to pass. Kindly fix this issue 👍

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/tests/editTask.test.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      },
+    ];
+  });
+
+  test('should clear all completed tasks', () => {
+    const updatedTasks = deleteCompletedTasks(tasks);
+    expect(updatedTasks.length).toEqual(1);
+    expect(updatedTasks[0].description).toEqual('Task 1');
+  });
+});
```
Comment (Gambit142) [inline src/tests/editTask.test.js]: - [ ] Great work writing these tests. Impressive work done 👏 To make your work complete, kindly write tests for the function that is responsible for changing the _**completed**_ status of a task. This means you should write tests for the function that changes a task's completed status from `true` to `false` and vice versa.

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @WilhelmK109 👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️ 
- Wrote tests for the function that edits task ✔️ 
- Wrote tests for the function that deletes all completed tasks ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @WilhelmK109    👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉

## Highlights
✔  Made required changes👍 
✔ All tests are passing👍 
✔ no linter errors 👍 

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**